test/image_test.py

```diff
@@ -1989,14 +1968,16 @@ def test_add_local_lazy_vs_copy(client, servicer, set_env_client, supports_on_pa
     assert all(fn.startswith("/root/pkg_a/") for fn in copied_files)
 
 
+def f():
```

Comment:
Very extra nit:

```suggestion
def noop():
```

(So it's clear from the test, the function does nothing)